atomics/T1562.002/T1562.002.yaml

```diff
@@ -20,4 +20,30 @@ atomic_tests:
       if(Test-Path "C:\Windows\System32\inetsrv\appcmd.exe"){
         C:\Windows\System32\inetsrv\appcmd.exe set config "#{website_name}" /section:httplogging /dontLog:false *>$null
       }
-    name: powershell
\ No newline at end of file
+    name: powershell
+
+- name: Kill Event Log Service Threads
+  description: Kill Event Log Service Threads using Invoke-Phant0m WARNING you will need to restart PC to return to normal state with Log Service Threads AND to run this again
```

Comment:
Please include the link to the blog post in the description as well. It is very helpful.